File: Daily Challanges/2026-05-May/2026-05-18.js
function getBingoRange(letter) {
  if (letter === "B") {
    return Array.from({ length: 15 }, (_, i) => i + 1);
  } else if (letter === "I") {
    return Array.from({ length: 15 }, (_, i) => i + 16);
  } else if (letter === "N") {
    return Array.from({ length: 15 }, (_, i) => i + 31);
  } else if (letter === "G") {
    return Array.from({ length: 15 }, (_, i) => i + 46);
  } else {
    return Array.from({ length: 15 }, (_, i) => i + 61);
  }
}

console.log(getBingoRange("B"));
console.log(getBingoRange("I"));
console.log(getBingoRange("N"));
console.log(getBingoRange("G"));
console.log(getBingoRange("O"));

File: Daily Challanges/2026-05-May/2026-05-18.py
def get_bingo_range(letter):
    if letter == "B":
        return list(range(1, 16))
    elif letter == "I":
        return list(range(16, 31))
    elif letter == "N":
        return list(range(31, 46))
    elif letter == "G":
        return list(range(46, 61))
    else:
        return list(range(61, 76))


print(get_bingo_range("B"))
print(get_bingo_range("I"))
print(get_bingo_range("N"))
print(get_bingo_range("G"))
print(get_bingo_range("O"))
